LongestSubstring_Test.java

```diff
@@ -0,0 +1,23 @@
+package Week1_Longest_Substring_without_repeat;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+import junit.framework.Assert;
+
+class LongestSubstring_Test {
+
+	@Test
+	void test() {
+	String s=null;

```

Comment:
You only set s to null, so its clearer just to put null in the method call instead of assigning it to a variable first